frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -1,21 +1,34 @@
 import { getSegmentBoundingBoxes, getSegmentVolumes } from "admin/admin_rest_api";
-import { Modal, Spin, Table } from "antd";
+import { Alert, Modal, Spin, Table } from "antd";
 import saveAs from "file-saver";
 import { formatNumberToVolume } from "libs/format_utils";
 import { useFetch } from "libs/react_helpers";
 import { Vector3 } from "oxalis/constants";
 import { getResolutionInfo } from "oxalis/model/accessors/dataset_accessor";
-import { Segment } from "oxalis/store";
+import { OxalisState, Segment } from "oxalis/store";
 import React from "react";
 import { SegmentHierarchyNode, SegmentHierarchyGroup } from "./segments_view_helper";
-import { Store, api } from "oxalis/singletons";
+import { api } from "oxalis/singletons";
 import { APISegmentationLayer } from "types/api_flow_types";
 import { voxelToNm3 } from "oxalis/model/scaleinfo";
 import { getBoundingBoxInMag1 } from "oxalis/model/sagas/volume/helpers";
+import { useSelector } from "react-redux";
+import {
+  getAdditionalCoordinatesAsString,
+  hasAdditionalCoordinates,
+} from "oxalis/model/accessors/flycam_accessor";
+
+const MODAL_ERROR_MESSAGE =
+  "Segment statistics could not not fetched. Check the console for more details.";
+const CONSOLE_ERROR_MESSAGE =
+  "Segment statistics could not not fetched due to the following reason:";
+const ERROR_CODE = CONSOLE_ERROR_MESSAGE;
 
 const SEGMENT_STATISTICS_CSV_HEADER =
   "segmendId,segmentName,groupId,groupName,volumeInVoxel,volumeInNm3,boundingBoxTopLeftPositionX,boundingBoxTopLeftPositionY,boundingBoxTopLeftPositionZ,boundingBoxSizeX,boundingBoxSizeY,boundingBoxSizeZ";
 
+const ADDITIONAL_COORDS_COLUMN = "addtionalCoordinates,";
```

Comment:
```suggestion
const ADDITIONAL_COORDS_COLUMN = "additionalCoordinates,";
```

Reply:
I'm not fond of the trailing comma in this string, because it's easy to miss and a bit unexpected (at least for me). I'd rather see the comma when concatting the column names.